src/commands/version.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+const packageConfig = require('../../package.json');
+
+/* Export as Yargs command */
+exports.command = 'version'
+exports.aliases = ['v']
+exports.desc = 'Show information about current orbit-db'
+
+exports.builder = function(yargs) {
```

Comment:
It doesn't say anything anywhere about style guidelines, so if you don't want to, you don't have to change this, but it'd be great to have `function() {}` as `() => {}`

Reply:
Actually I was following the `info.js` as per you guided, so basically I copied the code from there and mostly changed the bit required for `version.js`.
I will surely make the changes into arrow operator way, thanks for pointing it out.

Also I will add test for the same.

Reply:
Oops, sorry! 😄 That's my bad then. Will have to remember to fix it.

Reply:
:rofl: Well in that case you have to make changes in your every `builder` method of all commands. If you want I can create a single PR for the change through out.

Reply:
A separate PR would be great if you want to fix it! Thank you :)